docs/docs/lib/php.mdx

```diff
@@ -34,7 +34,7 @@ $growthbook = Growthbook\Growthbook::create()
 
 // Load feature flags from the GrowthBook API
 // Make sure to use caching in production! (see 'Loading Features' below)
-$growthbook->loadFeatures("sdk-abc123", "https://cdn.growthbook.io");
+$growthbook->initialize("sdk-abc123", "https://cdn.growthbook.io");
```

Comment:
Should we add a callout so existing users aren't confused?
> Note: Prior to version X, this method was called `loadFeatures` instead of `initialize`

Reply:
Agreed! Updated